README.rst

```diff
@@ -148,7 +148,7 @@ NI-DMM Python API Status
 +-------------------------------+----------------------+
 | NI-DMM (nidmm)                |                      |
 +===============================+======================+
-| Driver Version Tested Against | 19.1.0               |
+| Driver Version Tested Against | 19.5.0               |
```

Comment:
This is the version of the NI-DMM full installation, but in some cases the runtime alone is not revved up and may be older.

I think we need to list the runtime version here even if older than the "full SDK installation" version.

Reply:
Fixed